### ISSUE TITLE
损失函数问题

### ISSUE THREAD

**Issue by xxxxk12308** [XavierJiezou/D2LS#4]
作者您好，再次非常感谢您分享的工作！
我注意到代码中静态字典损失函数的实现仅包含 cross-entropy loss，没有使用论文中提到的 Dice loss，这和论文描述似乎不太一致。请问在复现时应该以论文中的设置为准，还是以目前公开的代码为准呢？
另外，还想请教一下，文中 FLOPs 的计算是否是基于输入尺寸 256 进行的？
感谢您的解答！

**Comment by caixiaoshun** [comment]
你好，感谢关注与反馈！

1. 关于损失函数
   在 `network/losses/useful_loss.py` 第 **76** 行，主损失定义为：
   `self.main_loss = JointLoss(SoftCrossEntropyLoss(...), DiceLoss(...), 1.0, 1.0)`
   也就是说训练时主分支采用 **CE + Dice** 的联合损失（权重均为 1.0，`smooth_factor=0.05`，并处理了 `ignore_index`）。另外在有辅助分支时，整体损失为：
   `loss = main_loss + 0.4 * aux_loss`，其中 `aux_loss` 为 `SoftCrossEntropyLoss`。因此实现与论文描述一致，复现请以 **当前开源代码** 为准。

2. 关于 FLOPs 计算
   FLOPs 的统计确实是基于 **输入尺寸 256（即 256×256）** 计算的。若使用其他分辨率，FLOPs 会随输入尺寸近似按平方比例变化。

**Comment by xxxxk12308** [comment]
非常感谢您的澄清。我在论文中看到您提到的静态损失是由 Dice 和 SoftCrossEntropyLoss 共同组成的。我的理解是，这部分对应于您代码中的 aux_loss，但实际上似乎仅采用了 SoftCrossEntropyLoss。不知道我的理解是否准确，还请您指正。

**Comment by caixiaoshun** [comment]
感谢指出这个问题！目前以仓库中的代码实现为准。再次感谢你的关注与反馈。
